opennms-doc/guide-admin/src/asciidoc/text/kafka-forwarder/opennms-kafka-forwarder.adoc

```diff
@@ -0,0 +1,55 @@
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+=== Kafka as data store for events/alarms/nodes
+Events, alarms and nodes from _{opennms-product-name}_ can be stored in kafka apart from storing in database.
+
+===== Enable kafka forwarder features
+
+Kafka forwarder is not enabled by default. Configure and enable features in karaf console.
+
+
+[source]
+----
+$ ssh -p 8101 admin@localhost
+...
+admin@opennms()> config:edit org.opennms.features.kafka.producer.client
+admin@opennms()> config:property-set bootstrap.servers 127.0.0.1:9092
+admin@opennms()> config:update
+admin@opennms()> feature:install kafka-streams
+admin@opennms()> feature:instal opennms-kafka-producer
+----
+Additional _kafka_ producer options can be set directly in the `org.opennms.features.kafka.producer.client.cfg` file in $OPENNMS_HOME/etc.
+A list of all the available options can be found here in link:https://kafka.apache.org/0100/documentation.html#producerconfigs[Producer Configs].
```

Comment:
We use version 1.0 of the client for this feature, so we should update the link to: https://kafka.apache.org/10/documentation.html#producerconfigs